keel-core/src/main/kotlin/com/netflix/spinnaker/keel/serialization/ObjectMappers.kt

```diff
@@ -19,17 +19,30 @@ import de.huxhorn.sulky.ulid.ULID
 import java.text.SimpleDateFormat
 import java.util.TimeZone
 
+class ObjectMapperSingleton private constructor() {
+  companion object {
+    val instance: ObjectMapper by lazy { ObjectMapper() }
+  }
+}
```

Comment:
You could literally just do ```private val objectMapperInstance by lazy { ObjectMapper().configureMe() }```

Reply:
You mean in place of the singleton class? But this is a module, not a class...

Reply:
Kotlin allows package level variables

Reply:
Ah! TIL.